.travis.yml

```diff
@@ -6,11 +6,11 @@
   matrix:
     include:
       - os: linux
-        go: 1.7
+        go: 1.7.x
```

Comment:
Why not "1.x"? It should be OK building with the latest Go and it won't need updating when new go is released?